js/ui/file_manager/ui.file_manager.js

```diff
@@ -314,6 +316,13 @@ class FileManager extends Widget {
             */
             fileProvider: null,
 
+            /**
+            * @name dxFileManagerOptions.path
```

Comment:
Path to what? Could you give an example?

Reply:
This is path to current directory that file manager displays.

Reply:
Shouldn't it be `currentPath`?

Reply:
Renamed.